### ISSUE TITLE
Denying fullscreen changes the state of player on youtube

### ISSUE THREAD

**Issue by luixxiul** [brave/browser-laptop#3918]
**Describe the issue you encountered:** Denying the fullscreen via notification bar on youtube changes the state of player into the fullscreen mode

**Expected behavior:** It should not change the state. Otherwise the state should be changed only after clicking "allow" on the notification bar

- Platform (Win7, 8, 10? macOS? Linux distro?): Windows 10

- Brave Version: 0.12.1

- Steps to reproduce:
    1. Open a video on youtube
    2. Click the fullscreen mode button on the player
    3. Click "Deny" on the notification bar

- Screenshot if needed:
![clipboard01](https://cloud.githubusercontent.com/assets/3362943/18442702/b9c96bc2-794c-11e6-9999-4368fe300611.png)

- Any related issues:

**Comment by luixxiul** [comment]
Setting a milestone for now.

**Comment by bbondy** [comment]
ya it's always been like this, it's annoying but we don't need to block on it.

**Comment by srirambv** [comment]
Duplicate of #3887

**Comment by srirambv** [comment]
Closing in favor of #3887
